server/lib/channel.dart

```diff
@@ -0,0 +1,39 @@
+import 'server.dart';
```

Comment:
is this a template file or from an example?

Reply:
It's a template file.